orc8r/cloud/go/services/orchestrator/obsidian/handlers/network_handlers_test.go

```diff
@@ -880,7 +880,7 @@ func Test_PutNetworkDNSHandlers(t *testing.T) {
 		ExpectedStatus: 400,
 		ExpectedError: "validation failure list:\n" +
 			"validation failure list:\n" +
-			"a_record.0 in body must be of type ipv4: \"192-88-99-142\"",
+			"records.0.a_record.0 in body must be of type ipv4: \"192-88-99-142\"",
```

Comment:
Should that be `dns.records.0.` as [above in line 319](https://github.com/magma/magma/pull/12151/files#diff-1e3584f3517597a3aa63e19a1e9c69b3f04fd065b144d6d61543353e8330dd25R319)

Reply:
Checked again an the other test indeed call a different URL. This one calls dns.